### ISSUE TITLE
Tests code coverage display for Twisted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11264]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#388|
|Type|enhancement|
|Created|2003-11-07 13:12:47Z|

<details><summary>Searchable metadata</summary>

```
trac-id__388 388
type__enhancement enhancement
reporter__itamarst itamarst
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__trial trial
keywords__ 
time__1068210767000000 1068210767000000
changetime__1469136313190702 1469136313190702
version__ 
owner__ 
cc__jml cc__dp
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml commented|
|-|-|

```
#!html
<pre>
Is this something distinct from --coverage?
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

I guess this was meant to be a viewer/browser for the output of --coverage.  Donovan once wrote something like this, I don't know what happened to it.

I don't think this belongs in Twisted, though.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset e74d32c3867e71936b1c3464280ced85994d06e0
```
#!CommitTicketReference repository="" revision="e74d32c3867e71936b1c3464280ced85994d06e0"
Merge pull request #388 from twisted/8647-acabhishek942-resourcewarning

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8647

Close more files
```
